### DISCUSSION TITLE
PR checks failing

### DISCUSSION THREAD

**Discussion by TheGiraffe3** [endless-sky/endless-sky#10511] [General]
Right now, all PRs which modify the copyright have failing checks because of something in the workflow script. It would be nice if that was fixed, as otherwise a whole bunch of PRs get failing checks for no reason.

Is there a way to easily do that?

**Comment by tibetiroka** [comment]
It's a known issue: https://discord.com/channels/251118043411775489/536900466655887360/1289041499232796683

We'll fix it after the release.

**Comment by TheGiraffe3** [comment]
#10522 has fixed this, so I'm closing it.
